Casks/m/mysql-shell.rb

```diff
@@ -1,23 +1,23 @@
 cask "mysql-shell" do
   arch arm: "arm64", intel: "x86-64bit"
 
-  version "8.2.1"
-  sha256 arm:   "099bb8194994f775190f9ef30a9c41192205c2ca04ea3566bb01e130edd96565",
-         intel: "117fcb004136227e2d6f74acc4a6bd42e1128fed6f8f4cc4180c278819435f4a"
+  version "8.3.0"
+  sha256 arm:   "9b772d1908c1413cafe1cd76eec4ae81014ab70ae820c87977396d584371b392",
+         intel: "a2121ba87bce1b65eb7f756107ea90b8d99e4c63d629987a8d4436ca28065fa7"
 
-  url "https://dev.mysql.com/get/Downloads/MySQL-Shell/mysql-shell-#{version}-macos13-#{arch}.dmg"
+  url "https://dev.mysql.com/get/Downloads/MySQL-Shell/mysql-shell-#{version}-macos14-#{arch}.dmg"
   name "MySQL Shell"
   desc "Interactive JavaScript, Python or SQL interface"
   homepage "https://dev.mysql.com/downloads/shell/"
 
   livecheck do
     url "https://dev.mysql.com/downloads/shell/?tpl=platform&os=33"
-    regex(/mysql[._-]shell[._-]v?(\d+(?:\.\d+)+)[._-]macos13[._-]#{arch}\.dmg/i)
+    regex(/mysql[._-]shell[._-]v?(\d+(?:\.\d+)+)[._-]macos14[._-]#{arch}\.dmg/i)
   end
 
   depends_on macos: ">= :monterey"
```

Comment:
```suggestion
  depends_on macos: ">= :ventura"
```

![Screenshot 2024-01-25 at 11 03 28 am](https://github.com/Homebrew/homebrew-cask/assets/40621599/92b54298-6c07-4c83-9194-7ebc46607428)

Reply:
Can we keep the older version which appears to support both Big Sur and Monterey along with coverage for the latest version that covers Ventura and Sonoma?  That way we can preserve some previous support.

Reply:
@krehel you mean keep the old 8.2.1, can you propose some change?

Reply:
Yes. Sure, I can fiddle with it.

Reply:
Pushed something to give this a try.